protos/protos/public/modeldb/versioning/VersioningService.proto

```diff
@@ -407,14 +407,13 @@ message MergeRepositoryCommitsRequest {
         repeated BlobDiff conflicts = 2;
         // In case of a conflict, stop at the first such conflict and return it
         Commit common_base = 3; // Common base shared between the two commits
-        //repeated BlobDiff diffs_a = 3; // Diff between the common base and commit A
-        //repeated BlobDiff diffs_b = 4; // Diff between the common base and commit B
     }
 }
 
 message RevertRepositoryCommitsRequest {
     RepositoryIdentification repository_id = 1;
-    string commit_sha_a = 2;//commit to revert
+    string commit_to_revert_sha = 2;//commit to revert
```

Comment:
changes to this file are the only intended changes, others are generated files

Reply:
Thank you for adding comments to the protos 😄

Reply:
the comment is literally _ replaced by spaces